lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
The other things that init to 0 tho end up assigned to numbers, right? ie, the convention is to initialize such that the type of the property doesn't change?